src/shared/sleep-config.c

```diff
@@ -421,6 +496,13 @@ static bool enough_swap_for_hibernation(void) {
         if (r < 0)
                 return false;
 
+        /* in cases where we were not able to identify available swap information, log a message with override instructions and fail */
+        if (r > 0 && hibernate_location->swap == NULL) {
+                log_debug("Unable to determine available swap space for resume=%s, resume_offset=%" PRIu64 ";set SYSTEMD_BYPASS_HIBERNATION_MEMORY_CHECK=1 to override this check.",
```

Comment:
This should be `log_warning`, otherwise most people will never see this information.

Reply:
Agreed. It would be great to make unable to check the space as a warning while confirmed insufficient space as error. After all if hibernation failed, the system should continue to function without corruption.